tangelo/toolboxes/ansatz_generator/uccsd.py

```diff
@@ -30,16 +30,17 @@
 
 import itertools
 import numpy as np
-from pyscf import mp
 from openfermion.circuits import uccsd_singlet_generator
 
 from tangelo.linq import Circuit
-
+from tangelo.helpers.utils import is_package_installed
 from .ansatz import Ansatz
 from .ansatz_utils import exp_pauliword_to_gates
 from ._unitary_cc_openshell import uccsd_openshell_paramsize, uccsd_openshell_generator, uccsd_openshell_get_packed_amplitudes
 from tangelo.toolboxes.qubit_mappings.mapping_transform import fermion_to_qubit_mapping
 from tangelo.toolboxes.qubit_mappings.statevector_mapping import get_reference_circuit
+from tangelo import SecondQuantizedMolecule
```

Comment:
can we try to reorder a bit all these tangelo imports ? It is a bit chaotic to read atm